src/Expirable.elm

```diff
@@ -115,9 +111,9 @@ in conjunction with `tickAll` to manage a list of `Expirable a`.
             Expirable.subscription DecrementToastMessages
 
 -}
-subscription : (Time.Time -> a) -> Sub a
+subscription : (Time.Posix -> a) -> Sub a
 subscription =
-    Time.every Time.second
+    Time.every 1000
```

Comment:
Thoughts on extracting `1000` to a constant function named something like `oneSecond` ?